lesson_4/customerForm.js

```diff
@@ -0,0 +1,52 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+  alert('!!!Увага!!! Усі дані треба вводити з заглавної літери!!!')
```

Comment:
This message is redundant. To avoid register problem you can use String method toLowerCase()
I'll add suggestion with example

Reply:
ok